src/components/search/advancedSearch/utils.js

```diff
@@ -0,0 +1,21 @@
+//if space between input add AND between
+export function converStateToCql({ inputFields }) {
+  if (!Array.isArray(inputFields) || inputFields.length === 0) {
+    return ""; // Return an empty string for an empty or invalid input.
+  }
+
+  const cqlQuery = inputFields.map((item) => {
+    const { value, prefixLogicalOperator, searchIndex } = item;
+    if (!value) {
+      return;
+    }
+    const cqlValue = value.split(" ").join(" AND ");
```

Comment:
Hvad hvis value er et array? (Tænker på dropdowns, som er en liste af values til ét search index :thinking:) 

Jeg er lidt bekymret for, om det her fungerer. F.eks. med følgende value:

```
prefixLogicalOperator = ""
searchIndex = "term.title"
value = "Harry Potter og de vises sten"
```
Bliver vist til:
`term.title = Harry AND Potter AND og AND de AND vises AND sten`.
Men "og de vises sten" kommer ikke på `term.title`, men i stedet for bare på frisøgning :thinking:  

Jeg tror at den cql som vi ønsker er noget i stil med:
`term.title = Harry AND term.title = Potter AND term.title = og AND term.title = de AND term.title = vises AND term.title = sten`

Reply:
Godt set. Det var jeg faktisk ikke opmærksom på at det virkede sådan 😬 har fikset det nu👏

Reply:
Mht. value. lige i den map kan er det kun for Inputfields. 

`export function convertStateToCql({ inputFields } = {}) {
`

`inputFields.map(value=>
`

når dropdowns er klar kan du bare tilføje dem som parametere til convertStateToCql 

`export function convertStateToCql({ inputFields,dropDowns } = {}) {
`